docs/hiring-guide/index.rst

```diff
@@ -0,0 +1,102 @@
+============
+Hiring Guide
```

Comment:
Great contribution!

When I read "Hiring Guide", I think this advice is limited to hiring managers and HR, so I'm trying to think of a different title. (and that's my only concern with the PR)

Maybe "Guide to Hiring and Getting Hired"?

Reply:
Or maybe "Hiring and Interviewing" (brainstorming -- I have an open mind on this)

Reply:
I was definitely wrestling with the question of the name too @mjang. I like your suggestion of "Guide to Hiring and Getting Hired" but perhaps it's a little long for the sidebar - how would you feel about keeping it the same in the sidebar but using the new title on the page itself?